CMakeLists.txt

```diff
@@ -364,13 +374,19 @@ if(HDF5_VOL_TEST_SERVER)
     )
   else()
     foreach(vol_test ${vol_tests})
+      set(last_vol_test "")
+
       add_test(NAME "h5vl_test_${vol_test}"
         COMMAND $<TARGET_FILE:h5vl_test_driver>
         --server ${HDF5_VOL_TEST_SERVER}
         --client $<TARGET_FILE:h5vl_test> ${vol_test}
         --serial
         ${HDF5_VOL_TEST_DRIVER_EXTRA_FLAGS}
       )
+
+      set_tests_properties("h5vl_test_${vol_test}" PROPERTIES DEPENDS "${last_vol_test}")
+
+      set(last_vol_test "h5vl_test_${vol_test}")
```

Comment:
Fixes an issue with concurrent testing of the main VOL tests when the "part testing" option is enabled. This should fix most of the testing with other VOLs.